src/ucp/wireup/select.c

```diff
@@ -208,6 +208,8 @@ ucp_wireup_test_select_flags(const ucp_wireup_select_flags_t *select_flags,
                              uint64_t flags, const char **flag_descs,
                              ucs_string_buffer_t *missing_flags_str)
 {
+    ucs_string_buffer_reset(missing_flags_str);
```

Comment:
IMO better reset before calling the function from ucp_wireup_select_transport
cc @shasson5

Reply:
fixed